cogs/hl.py

```diff
@@ -30,9 +30,20 @@ class Highlighter(AceMixin, commands.Cog):
 
 	@commands.command(aliases=['h1'])
 	@commands.bot_has_permissions(manage_messages=True, add_reactions=True)
-	async def hl(self, ctx, *, code):
+	async def hl(self, ctx, *, code: str = None):
 		'''Highlight some code.'''
-
+		if ctx.message.reference is not None:
+			# get the replied to message contents if there is one
+			ref_msg = ctx.message.reference.resolved
+			if ref_msg.author.bot:
+				raise commands.CommandError('Can not paste bot messages.')
+			code = ref_msg.content
+		else:
+			# include spaces/tabs at the beginning
+			code = ctx.message.content[len(ctx.prefix) + 3:]
+			if len(code) == 0:
+				raise commands.CommandError('No code to paste.')
+		
```

Comment:
```suggestion
		if code is not None:
			# get the replied to message contents if there is one
			ref_msg = ctx.message.reference.resolved
			if ref_msg.author.bot:
				return
			code = ref_msg.content
```
You're never actually using the parameter `code`.